cpp/src/io/parquet/reader_impl.cu

```diff
@@ -411,6 +411,9 @@ class aggregate_metadata {
     // walk upwards, skipping repeated fields
     while (schema_index > 0) {
       if (!pfm.schema[schema_index].is_stub()) { depth++; }
+      // schema of one-level encoding list doesn't contain nesting information, so we need to
+      // manually add an extra nesting level
```

Comment:
what a weird piece of specs

Reply:
Agreed. It's a flat schema representing a nested data structure.